dotnet/samples/Concepts/ChatCompletion/MistralAI_FunctionCalling.cs

```diff
@@ -0,0 +1,202 @@
+        const string ChatPrompt = @"
+            <message role=""user"">What is the weather like in Paris?</message>
+        ";
+        var executionSettings = new MistralAIPromptExecutionSettings
+        {
+            ToolCallBehavior = MistralAIToolCallBehavior.RequiredFunctions(plugin, true)
+        };
+        var chatSemanticFunction = kernel.CreateFunctionFromPrompt(
+            ChatPrompt, executionSettings);
+        var chatPromptResult = await kernel.InvokeAsync(chatSemanticFunction);
+
+        WriteLine(chatPromptResult);
+    }
+
+    [Fact]
+    public async Task NoKernelFunctionsAsync()
+    {
+        // Create a logging handler to output HTTP requests and responses
+        var handler = new LoggingHandler(new HttpClientHandler(), this.Output);
+        HttpClient httpClient = new(handler);
+
+        // Create a kernel with MistralAI chat completion and WeatherPlugin
+        IKernelBuilder kernelBuilder = Kernel.CreateBuilder();
+        kernelBuilder.AddMistralChatCompletion(
+                modelId: TestConfiguration.MistralAI.ChatModelId!,
+                apiKey: TestConfiguration.MistralAI.ApiKey!,
+                httpClient: httpClient);
+        kernelBuilder.Plugins.AddFromType<WeatherPlugin>();
+        Kernel kernel = kernelBuilder.Build();
+
+        // Invoke chat prompt with auto invocation of functions enabled
+        const string ChatPrompt = @"
+            <message role=""user"">What is the weather like in Paris?</message>
+        ";
+        var executionSettings = new MistralAIPromptExecutionSettings
+        {
+            ToolCallBehavior = MistralAIToolCallBehavior.NoKernelFunctions
+        };
+        var chatSemanticFunction = kernel.CreateFunctionFromPrompt(
+            ChatPrompt, executionSettings);
+        var chatPromptResult = await kernel.InvokeAsync(chatSemanticFunction);
+
+        WriteLine(chatPromptResult);
+    }
+
+    [Fact]
+    public async Task AutoInvokeKernelFunctionsMultiplePluginsAsync()
+    {
+        // Create a logging handler to output HTTP requests and responses
+        var handler = new LoggingHandler(new HttpClientHandler(), this.Output);
+        HttpClient httpClient = new(handler);
+
+        // Create a kernel with MistralAI chat completion and WeatherPlugin
+        IKernelBuilder kernelBuilder = Kernel.CreateBuilder();
+        kernelBuilder.AddMistralChatCompletion(
+                modelId: TestConfiguration.MistralAI.ChatModelId!,
+                apiKey: TestConfiguration.MistralAI.ApiKey!,
+                httpClient: httpClient);
+        kernelBuilder.Plugins.AddFromType<WeatherPlugin>();
+        kernelBuilder.Plugins.AddFromType<WidgetFactory>();
+        Kernel kernel = kernelBuilder.Build();
+
+        // Invoke chat prompt with auto invocation of functions enabled
+        const string ChatPrompt = @"
+            <message role=""user"">Create a lime and scarlet colored widget for me.</message>
```

Comment:
Use `"""` literals for better legibility

```suggestion
        const string ChatPrompt = """<message role="user">Create a lime and scarlet colored widget for me.</message>"""
```